nginx/conf.d/server/server.api

```diff
@@ -246,6 +246,11 @@ location /skynet/skyfile {
         if skynet_modules.is_enabled("s") then
             skynet_scanner.scan_skylink(ngx.header["Skynet-Skylink"])
         end
+
+        if skynet_modules.is_enabled("p") then
+            local skynet_pinner = require("skynet.pinner")
+            skynet_pinner.pin_skylink(ngx.header["Skynet-Skylink"])
```

Comment:
> We should really add comments here explaining what's going on. This one, for example, should be:
> 
> This basically goes for every single call to pinner here, so I won't repeat it.

added proper comments in https://github.com/SkynetLabs/webportal-nginx/pull/2/commits/49c0d46345b2ebae4b5403035907ca452622d32c

Reply:
> I would also heavily advocate for extensive comments on `skynet_tracker` and how and why is it different from `skynet_account` because from looking at those two I can tell why they are separate.

this pull request doesn't really affect tracker nor account so I've created follow up issue to improve docs on lua modules overall https://github.com/SkynetLabs/webportal-nginx/issues/5